module/VuFind/src/VuFind/Db/Entity/UserCard.php

```diff
@@ -38,75 +38,107 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="user_card", indexes={@ORM\Index(name="user_card_cat_username", columns={"cat_username"}), @ORM\Index(name="user_id", columns={"user_id"})})
+ * @ORM\Table(name="user_card",
+ * indexes={@ORM\Index(name="user_card_cat_username", columns={"cat_username"}),
+ * @ORM\Index(name="user_id",   columns={"user_id"})})
  * @ORM\Entity
  */
 class UserCard implements EntityInterface
 {
     /**
+     * Unique ID.
+     *
      * @var int
      *
-     * @ORM\Column(name="id", type="integer", nullable=false)
+     * @ORM\Column(name="id",
+     *          type="integer",
+     *          nullable=false
+     * )
      * @ORM\Id
      * @ORM\GeneratedValue(strategy="IDENTITY")
      */
     protected $id;
 
     /**
+     * Card name.
+     *
      * @var string
      *
      * @ORM\Column(name="card_name", type="string", length=255, nullable=false)
      */
     protected $cardName = '';
 
     /**
+     * Cat username.
+     *
      * @var string
      *
      * @ORM\Column(name="cat_username", type="string", length=50, nullable=false)
      */
     protected $catUsername = '';
 
     /**
+     * Cat password.
+     *
      * @var string|null
      *
      * @ORM\Column(name="cat_password", type="string", length=70, nullable=true)
      */
     protected $catPassword;
 
     /**
+     * Cat password encryption.
```

Comment:
I'd change this to `Cat password (encrypted)` -- the way VuFind works, depending on settings, the value may be stored in plain text or encrypted, and different columns are used for each scenario.